src/app/cluster/cluster-details/alertmanager-config/template.html

```diff
@@ -0,0 +1,36 @@
+<!--
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<div *ngIf="!isLoadingData()"
+     class="header">
+  <div fxFlex></div>
+  <button id="km-alertmanager-config-reset-btn"
+          mat-icon-button

```

Comment:
With the latest redesign the styling of these buttons was changed. Can you adjust?

Reply:
done 👍